### ISSUE TITLE
any video tutorial?

### ISSUE THREAD

**Issue by abcnow** [shamblett/alpaca#8]
@shamblett Man.. thanks a lot for taking the task to do this! I wonder if you can do a simple video tutorial on how to install and use this package.. thanks in advance!

**Comment by shamblett** [comment]
Its not something I'd thought of but I'll consider it if it helps others.

This is not a package as such as its not been published to pub, you just pull the repo onto your linux machine and follow the instructions in the README. If you find anything wrong/confusing/needs clarification in the README please post it here and I'll see what I can do.

**Comment by abcnow** [comment]
Hi @shamblett .. thanks a lot man for your note.. I was tired yesterday.. I read your notes today and they are very clear.. I think your code should get more attention.. there are so many things you can use this for and youtube is the best way to get the word around.. keep it up.. if you need a good design for this.. just let me know..

**Comment by shamblett** [comment]
Yep I agree, its always hard to get stuff like this into the right hands like I say I will look at making a video for this.

Glad you got it working.
